### DISCUSSION TITLE
Can I use Vortice in .NET Framework Winforms projects?

### DISCUSSION THREAD

**Discussion by bulent017** [amerkoleci/Vortice.Windows#435] [General]
Can I use Vortice in .NET Framework Winforms projects?

Because I am getting this erorr inmy project:
System.IO.FileNotFoundException: ''Vortice.Direct3D11, Version=2.3.0.0, Culture=neutral, PublicKeyToken=5431ec61a7e925da' It failed to load the file or its assembly code or one of its dependencies. The system cannot find the specified file.'

I can't add it to References.

**Comment by MarcioAB** [comment]
Can I use Vortice in .NET Framework Winforms projects?
**YES**

**Comment by MarcioAB** [comment]
The current NuGet version is 3.5.0
![image](https://github.com/amerkoleci/Vortice.Windows/assets/15105477/6691925d-c77e-4f18-962d-3e06954dec3b)

![image](https://github.com/amerkoleci/Vortice.Windows/assets/15105477/71f66e1f-0bd0-4cf7-ae38-d60f0f8086f4)

**Comment by Revan1985** [comment]
For usage with .net framework (<= 4.8), you need to target the version 2.4.2 for nuget package.
from 3.2 support for netstandard has been dropped, only .net6+ (7+ from last versions)
